### ISSUE TITLE
Unable to create a new session in Redis

### ISSUE THREAD

**Issue by cromedome** [burnersk/Dancer2-Session-Redis#8]
Any attempt to use Redis as a session engine with this plugin yields the following error:

`Unable to create a new session: [expire] ERR Protocol error: invalid bulk length,  at /Users/jason/.plenv/versions/vm-5.24.0/lib/perl5/site_perl/5.24.0/Dancer2/Session/Redis.pm line 152. at /Users/jason/.plenv/versions/vm-5.24.0/lib/perl5/site_perl/5.24.0/Dancer2/Core/App.pm line 434.`

I have tried with and without a serializer and get the same results.

My config looks like this:
```
session: "Redis"
 
engines:
    session: 
        Redis:
            cookie_name: vm
            redis_server: "localhost:6379"
            redis_serialization:
                module: "Dancer2::Session::Redis::Serialization::Sereal"
                compression: "snappy"
```

Any ideas? Thanks!

**Comment by SysPete** [comment]
@cromedome I've cooked up a branch with lots of debug:

https://github.com/burnersk/Dancer2-Session-Redis/tree/issues/gh-8-debug

Any chance you could try it? You'll get a *lot* of debug logged especially if you are storing a lot of data in the session.

**Comment by SysPete** [comment]
So it seems that `session_duration` is a required config option for this session engine. I'll add checks for this in the code and cook a new release tomorrow.

**Comment by SysPete** [comment]
@cromedome this is fixed in latest release v0.004 just uploaded to CPAN.
